testsuites/sparql/src/main/java/org/eclipse/rdf4j/testsuite/sparql/RepositorySPARQLComplianceTestSuite.java

```diff
@@ -81,12 +81,12 @@ public static void tearDownFactory() throws Exception {
 		ValuesTest.class, OrderByTest.class, ExistsTest.class, MinusTest.class })
 @Experimental
 public abstract class RepositorySPARQLComplianceTestSuite {
-	@BeforeClass
+	@BeforeAll
```

Comment:
I think that this abstract suite (and its concrete subclasses) is one of the few tests where we _can't_ upgrade to JUnit 5 yet: it uses the JUnit 4 `Suite` runner, which (last time I looked) Junit 5 had no comparable feature for (or at least not something I could get to work)...

Reply:
Yes, I am realizing it might not be possible to fully migrate to JUnit 5 in this PR. But I will make an effort, and comment on the few places where we are stuck with JUnit 4 for now. 😅

Reply:
Reverted this upgrade as suggested.